bin/csv_to_krun_json

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env pypy
```

Comment:
I don't think this script is particularly slow? If so, we should probably just use `python2.7` (or whatever version of CPython is capable of running this), simply because more people are likely to have it installed already.